pwnlib/libcdb.py

```diff
@@ -126,7 +127,23 @@ def provider_local_system(hex_encoded_id, hash_type):
         return local_libc.data
     return None
 
-PROVIDERS = [provider_local_system, provider_libcdb, provider_libc_rip]
+# Offline search https://github.com/niklasb/libc-database for hash type
+def provider_local_database(hex_encoded_id, hash_type):
+    assert context.local_libcdb
```

Comment:
What do you assert here? Isn't the check below enough? Maybe add a check for None/empty string below instead.